### PR TITLE
Add blog: Learn Scala with Online Courses

### PR DESCRIPTION
**Pull request by julienrf**
Screenshot 2022-01-18 at 10-20-00 Learn Scala

### CODE REVIEW

**Review by vincenzobaz — APPROVED** [review]

**Review by SethTisue — APPROVED** [review]
LGTM but you might consider coming up with a more specific title, one that indicates that the post is about the courses specifically, not about learning Scala in general.

**Comment by julienrf** [comment]
Thank you Seth. I changed for “Learn Scala with Online Courses”. How does that sound?

File: blog/_posts/2022-01-19-learn-scala-with-online-courses.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+We also kept the Scala 2 versions of these courses online, but we observed 
+that the majority of the learners took the Scala 3 versions (e.g., 97% in the 
+case of Functional Programming Principles in Scala).
+
+The only courses that still use Scala 2 are the courses that depend on Spark,
+which does not yet fully support Scala 3 (see below our plans in this regard).
+
+We also published a new course: [Effective Programming in Scala]. This 
+course teaches programmers all they need to be ready to work in Scala.
```
Comment (sjrd) [inline blog/_posts/2022-01-19-learn-scala-with-online-courses.md:37]:
```suggestion
course teaches programmers everything they need to be ready to work in Scala.
```

**Review by sjrd — APPROVED** [review]

File: blog/_posts/2022-01-19-learn-scala-with-online-courses.md (hunk 1 above)
Comment (julienrf) [reply blog/_posts/2022-01-19-learn-scala-with-online-courses.md:37]: Done, and squashed.

**Comment by SethTisue** [comment]
👍

**Pull request merged**